docs/user-guide/time-and-costs/progress-tracking/README.md

```diff
@@ -2,93 +2,111 @@
 
--  the % Complete value from the work package progress itself, set manually (on the left)
-- the value, calculated as an aggregation of % Complete values of the children work packages. This calculated value is a hyperlink, clicking it will open the detailed view of the children and illustrate the computation.
+| **Old term**   	 | **New term**   	      |
+|--------------------|------------------------|
+| Progress       	 | %&nbsp;Complete	      |
+| Estimated time 	 | Work           	      |
+| Remaining time 	 | Remaining work 	      |
 
-![](progress-tracking-two-progress-values.png)
+>**Note**: You will still find the new attributes if you search using their older names (in the list of filters, for example).
 
-## Manual progress tracking
+## Progress reporting modes
 
-| Topic                                                                                               | Content                                           |
-|-----------------------------------------------------------------------------------------------------|:--------------------------------------------------|
-| [Manual progress tracking](#manual-progress-tracking)                                               | How to track progress manually per work package.  |
-| [Progress tracking in the work package hierarchy](#progress-tracking-in-the-work-package-hierarchy) | How to track progress for multiple work packages. |
-| [Status based progress tracking](#status-based-progress-tracking)                                   | How to track progress use work package status.    |
+OpenProject offers two modes for reporting progress:
 
-## Manual progress tracking
+- **Work-based progress reporting** enables you to automatically derive progress based on the values you enter for Work and Remaining work
+- **Status-based progress reporting** allows you to assign fixed % Complete values to statuses, and automatically derive Remaining work based on the values for Work you can enter
 
-After the initial installation, OpenProject is configured for manual progress tracking. In order to log progress,
-please open the details of a work package.
-The **% Complete (earlier called Progress (%))** field shows the default value of 0%.
+>**Note:** The administrator of your instance will have selected a mode for the entire instance. If you are an administrator, you can modify this by following our [admin guide on work package settings](../../../system-admin-guide/manage-work-packages/work-package-settings).
 
-![Work package progress field](progress-tracking-wp-field.png)
+### Work-based progress reporting
 
-Clicking on the progress bar opens an in-place editor
-that allows you to enter a percentage value between 0 and 100.
+%&nbsp;Complete is an automatically calculated value that is a function of Work and Remaining work (unless %&nbsp;Complete is configured to be [set by status](#status-based-progress-reporting).
```

Comment:
```suggestion
%&nbsp;Complete is an automatically calculated value that is a function of Work and Remaining work (unless %&nbsp;Complete is configured to be [set by status](#status-based-progress-reporting)).
```
Missing closing paren. 
Could also be rewritten without parens and just a comma.